README.md

```diff
@@ -218,6 +219,50 @@ Note that First-Party Sets also gives browsers the opportunity to group per-site
 those at `chrome://settings/content/all`) by the “first-party” boundary instead of eTLD+1, which is 
 not always the correct site boundary.
 
+# Clearing Site Data on Set Transitions
+Sites can switch the First-Party Set that they are a member of. We need to pay attention to these transitions so that they don’t link user identities across all the FPSs they’ve historically been in. The concrete goals that we aim to achieve are as follows:
+
+*   Preserve user expectations that their browsing activity is scoped/partitioned to “party” (as defined by First-Party Set).
+*   Prevent a potential privacy attack that uses a throwaway domain to rotate between sets, and therefore sync user data or identifiers across disparate sets. 
+
+In order to achieve this, site data needs to be cleared on certain transitions., The clearing should behave like [Clear-Site-Data](https://www.w3.org/TR/clear-site-data/), for which site data includes cookies, storage, cache, as well as execution contexts (documents, workers, etc.). We don’t differentiate between different types of site data since a site could expect different types of data to co-exist and require some consistency.
```

Comment:
> , and because a user identifier could be stored in any of these storage areas.

Instead of adding the sentence in the end, how about: 
We don't differentiate between different types of site data because a user identifier could be stored in any of these storage areas, so a site could expect different types of data to co-exist and require some consistency.
?

Reply:
Your phrasing isn't quite right: user identifiers aren't the only reason a site might expect data in different areas to be consistent, so it's not a "so" relationship. We could use

> We don’t differentiate between different types of site data because:
> * A user identifier could be stored in any of these storage types.
> * Clearing just a few of the types would break sites that expect different types of data to be consistent with each other.

?

Reply:
SG!